misc/maven-resolver/src/test/java/org/ballerinalang/maven/MavenResolverTest.java

```diff
@@ -52,4 +52,33 @@ public void testTransitiveDependency() {
             Assert.fail(e.getMessage());
         }
     }
+
+    @Test
+    public void testAddRemoteRepository() {
+        try {
+            resolver.addRepository("wso2-releases", "http://maven.wso2.org/nexus/content/repositories/releases/");
+            Dependency dependency = resolver.resolve("org.ballerinalang", "wso2-cassandra",
+                    "0.8.3", true);
+            String jarPath = Utils.getJarPath(targetRepo, dependency.getDepedencies().get(0));
+            Assert.assertTrue(new File(jarPath).exists());
+        } catch (MavenResolverException e) {
+            Assert.fail(e.getMessage());
+        }
+    }
+
+    @Test(enabled = false)
```

Comment:
Why have we disabled this test?

Reply:
Credentials are not added. Thus, it is temporarily disabled.